.zuul.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 - job:
-    name: functional-autoscaling-tests-osp18
+    name: functional-autoscaling-graphing-tests-osp18
```

Comment:
IMHO, there is no need to rename this job because of the extra tests being added. 
This removes the need to edit telemetry-operator to update the job name